@here/harp-examples/src/markers_dynamic.ts

```diff
@@ -189,8 +189,10 @@ export namespace DynamicMarkersExample {
     function addInstructions() {
         const message = document.createElement("div");
         message.innerHTML = `Tap map to add a marker, long press on a marker to remove it`;
-        message.style.position = "relative";
+        message.style.position = "absolute";
```

Comment:
For the sake of consistency, I recommend using the same styling for the **info** element as in ``datasource_features_lines-and-points`` example.